runners/spark/src/main/java/org/apache/beam/runners/spark/examples/WordCount.java

```diff
@@ -54,7 +55,7 @@ public void processElement(ProcessContext c) {
       }
 
       // Split the line into words.
-      String[] words = c.element().split("[^\\p{L}]+");
+      Iterable<String> words = Splitter.onPattern("[^\\p{L}]+").split(c.element());
```

Comment:
Is this a error-prone reported issue ? This class is just copied from the examples wordcount one and ideally should not differ from it (not also introduce guava in its dependencies).

Reply:
Mmm I se ethis i s clearly a warning on error-prone, can you somehow ignore it.